assets/js/partials/site_scripts.js

```diff
@@ -1,20 +1,22 @@
 /**
  * Sticky header
+ * mispell as JS comment
  */
 
-jQuery(document).on("scroll", function() {
+jQuery(document).on("scroll", function () {
```

Comment:
🚫 **[phpcs]** <PEAR.Functions.FunctionCallSignature.MultipleArguments> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Only one argument is allowed per line in a multi-line function call